---
title: 'Blaster'
tags:
  - sequence
  - blast
  - bowtie
authors:
 - name: Stefanie Lueck
   orcid: 0000-0003-0536-835X
   affiliation: 1
 - name: Dimitar Douchkov
   orcid: 0000-0001-6603-4930
   affiliation: 1
affiliations:
 - name: Leibniz-Institut für Pflanzengenetik und Kulturpflanzenforschung Gatersleben, Stadt Seeland, Sachsen-Anhalt
   index: 1
date: 30 March 2019
bibliography: paper.bib
---

# Summary

Common sequence alignment tools such as Basic Local Alignment Search Tool (BLAST) or Bowtie short sequence aligner are used by scientist on a daily basis. Often those programs are with command line interface which makes them very practical for developers and advance users but hard for researchers who are more familiar with a graphical user interface. Here we present a tool called “Blaster”, which offers simple and intuitive graphical interface for performing some of the most common tasks in nucleotide and protein sequence work. The software is provided with the following modules:
·        Sequence similarity search (blastn, blastp, blastx, tblastn, tblastx, Bowtie) in local databases
·        Multiple sequence alignment (CLUSTALW2)
·        Sequence extraction from multiple FASTA files by sequence ID
·        Sequence converter (reverse complement, reverse, translate)
·        Sequence format conversions (FASTA, Multiple FASTA, FASTQ, CSV)
·        Local  sequence database management
“Blaster” is available as Microsoft Windows installer and the open source code is accessible on GitHub with xx license.


This is a proof of concept integration between a GitHub [@GitHub] repo and figshare [@figshare] in an effort to get a DOI for a GitHub repository. When a repository is tagged for release on GitHub, Fidgit [@Fidgit] will import the release into figshare thus giving the code bundle a DOI. In a somewhat meta fashion, Fidgit is publishing itself to figshare with DOI 'https://doi.org/10.6084/m9.figshare.828487' [@figshare_archive].

-![Fidgit deposited in figshare.](figshare_article.png)

# References
